formula/src/main/java/com/instacart/formula/RuntimeExtensions.kt

```diff
@@ -91,7 +87,6 @@ private fun <Input : Any, Output : Any> start(
     val callbackFlow = callbackFlow {
         val runtime = formula.asRuntime(coroutineContext, config)
         runtime.setOnOutput(this::trySend)
-        runtime.setOnError(this::close)
```

Comment:
There is no need to emit the error as we rely on the global error listener instead.